bridges/snowbridge/pallets/ethereum-client/src/tests_electra.rs

```diff
@@ -0,0 +1,926 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023 Snowfork <hello@snowfork.com>
+pub use crate::mock_electra::*;
```

Comment:
Full test suite for Electra. These tests can be moved to `tests.rs` once Electra has been merged, and this file can be deleted. The same applies to `mock_electra.rs`.